pallets/identity/src/lib.rs

```diff
@@ -633,6 +633,24 @@ decl_module! {
             }
         }
 
+        /// Assuming this is executed by the GC voting majority, adds a new cdd claim record.
+        #[weight = 950_000_000]
+        pub fn gc_add_cdd_claim(
+            origin,
+            target: IdentityId,
+            expiry: Option<T::Moment>,
+        ) -> DispatchResult {
+            T::GCVotingMajorityOrigin::ensure_origin(origin)?;
+            Self::base_add_cdd_claim(target, Claim::make_cdd_wildcard(), GC_DID, expiry)
+        }
+
+        /// Assuming this is executed by the GC voting majority, removes an existing cdd claim record.
+        #[weight = 500_000_000]
+        pub fn gc_revoke_cdd_claim(origin, target: IdentityId) {
+            T::GCVotingMajorityOrigin::ensure_origin(origin)?;
+            Self::base_revoke_claim(target, ClaimType::CustomerDueDiligence, GC_DID, None)
+        }
```

Comment:
It is better to add last in the list of dispatchable.

Reply:
Thanks, fixed.